TestCases.xml

```diff
@@ -962,14 +962,14 @@
         <Requests>
           <!-- Set up the test -->
           <Lock Lock="LockString" />
-          <PutFile Lock="LockString" ResourceId="ZeroByteFile" />
+          <PutFile Lock="LockString" ResourceId="WordSimpleDocument" />
```

Comment:
This will break the test. The test is testing this part of the PutFile spec: "When a host receives a PutFile request on a file that is not locked, the host must check the current size of the file. If it is 0 bytes, the PutFile request should be considered valid and should proceed. If it is any value other than 0 bytes, or is missing altogether, the host should respond with a 409 Conflict."

In order to test that, the test needs to ensure that the file is zero bytes. So the test does the following:

1. Locks the file
2. Executes PutFile with a zero-byte file; this request includes the lock header set to the lock that was taken in step 1.
3. Unlocks the file - now the file is zero bytes and unlocked
4. Calls PutFile without a lock header, expecting it to succeed since the target file is zero bytes
5. GetFile to confirm the file has been updated
6. Unlock to try and clean up the lock (this is just for error cases - we try to clean up in every test just in case)

Steps 1-3 are just setting up the test, and steps 4-5 are the actual test.

Reply:
Thank you for answer me!
You are right! my bad, i followed the PnP-WOPI implementation
https://github.com/OfficeDev/PnP-WOPI/blob/master/com.microsoft.dx.officewopi/Utils/WopiExtensions.cs line 381 by checking the incoming file.
But i need to check the CURRENT_FILE. I got it now!